### ISSUE TITLE
No Focus Visible - "Attach Files" Button

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3475]
**Issue Description**: There is no visible focus outline on the "Attach files" button in the chat window.
<img width="899" alt="Screenshot 2024-07-29 at 2 55 01 PM" src="https://github.com/user-attachments/assets/a7a135d3-de18-4c10-a9d9-d4bb43f80c58">

**WCAG Criteria**: [2.4.7 Focus Visible - AA](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible)

**Screenshot or Snippet**: [Link to Screenshot](https://share.zight.com/DOuEpDGE)

**Notes**: Visible focus outline assists keyboard  and other non-mouse users in navigating the interface effectively.

**Comment by danny-avila** [comment]
I believe this was resolved in #3536 
![image](https://github.com/user-attachments/assets/b742ff3e-8f52-41d1-841e-3e5f521d8a9f)
![image](https://github.com/user-attachments/assets/696b8930-9a6e-49ab-a558-a2419142e55c)
